src/pages/Account/Profile.tsx

```diff
@@ -66,36 +66,43 @@ const Profile: React.FunctionComponent<PropsType> = ({ location }) => {
     <div className="page-container-onboarding">
       <div className="login-container">
         <h2 className="login-welcome-title">
-          Well done! Now create your profile
+          Hey, Mona. We’re so glad you’re here! Upload or take a photo and share
+          your Summit snap here.
         </h2>
         <div className="login-welcome-subtitle">
-          {`Don't fret, you'll be able to edit it at any time later`}
+          {`Swing back and edit your profile anytime you like.`}
```

Comment:
[nit] design
![image](https://user-images.githubusercontent.com/63313732/122396633-2d52e600-cf46-11eb-8248-04e2c0e960cf.png)

our version
![image](https://user-images.githubusercontent.com/63313732/122396684-3ba10200-cf46-11eb-8249-47a0e7fbf267.png)

looks like we need to give a bigger `margin-top` to the "Swing back..." text